### ISSUE TITLE
📃 Iteration 27 roadmap 📃

### ISSUE THREAD

**Issue by mlewand** [ckeditor/ckeditor5#2004]
This post outlines our targets for the upcoming iteration. The goal is to provide a summary of where we are going.

## New features

* [Inserting a horizontal line](https://github.com/ckeditor/ckeditor5/issues/1365) - Merged to the master branch 🎉
* [Inserting a page break](https://github.com/ckeditor/ckeditor5/issues/1194) - Merged to the master branch 🎉
* [Responsive toolbar](https://github.com/ckeditor/ckeditor5/issues/416#issuecomment-430246472) - Merged to the master branch 🎉
* [Title plugin](https://github.com/ckeditor/ckeditor5/issues/2003) - Merged to the master branch 🎉
* [Improved bold retention](https://github.com/ckeditor/ckeditor5-basic-styles/issues/94), this will also fix [pasting bold from Google Docs](https://github.com/ckeditor/ckeditor5-paste-from-office/issues/74) - Merged to the master branch 🎉
* [SVG upload support](https://github.com/ckeditor/ckeditor5/issues/674) - Merged to the master branch 🎉
	* [Other formats like webp, tiff too](https://github.com/ckeditor/ckeditor5-image/issues/295) - Merged to the master branch 🎉
* [`.sketch` file for CKEditor 5 theme](https://github.com/ckeditor/ckeditor5/issues/673) - Published 🎉

## Collaborative features

* Add support for image resize in track changes

## Bug fixes

* Minor RTL tweaks
* Minor image resizer tweaks

We are also planning to simplify our issue handling. We're moving away from having a separate issue tracker for each repository. [All the tickets will be kept in the main CKEditor 5 repository](https://github.com/ckeditor/ckeditor5/issues/2002) instead. While doing so we'll ensure that all the issues are transferred correctly, so no reports will be lost in the process.

Finally, there are more minor issues planned for the 27th iteration. See the [list of issues assigned to the current iteration](https://github.com/ckeditor/ckeditor5/issues?utf8=%E2%9C%93&q=is%3Aissue+milestone%3A%22iteration+27%22+-label%3Atype%3Adocs+-label%3Atype%3Atask+) for more details.

## Dates

The code freeze is planned to happen between the 16th and 18th of October, followed by releasing npm packages on October the 23rd. Social followup will happen between 23rd and 24th of October.

Please keep in mind that these plans are subject to change and even though we put all our energy to deliver all the items mentioned above, the iteration plan may change at any point.

**Comment by Victz** [comment]
Any update for code snippet ? 🙏

**Comment by totorean** [comment]
Great work guys! Waiting on Horizontal rule woop woop

**Comment by mlewand** [comment]
[The horizontal rule feature](https://github.com/ckeditor/ckeditor5/issues/1365) has just been merged 🎉

**Comment by mlewand** [comment]
Meanwhile we also merged the [title](https://github.com/ckeditor/ckeditor5/issues/2003) plugin 🎉

**Comment by mlewand** [comment]
There's a preview of [page break](https://github.com/ckeditor/ckeditor5/issues/1194#issuecomment-535524916) plugin available 🔥 Feel free to give it a try.

**Comment by mlewand** [comment]
@Victz, all, we regret to inform you that due to some unforeseen absences in the development team we will not be able to deliver code blocks in this release 😞

That being said, we know that you want to have it delivered as soon as possible (so do we!) and we’re going to ship it in the next iteration. Stay tuned!

**Comment by mlewand** [comment]
We also published [a .sketch file for CKEditor 5 theme](https://www.sketchappsources.com/free-source/4125-ckeditor-5-rich-text-editor-sketch-freebie-resource.html) :tada: So if you'd like to play with our theme, there is no better time to do that!

**Comment by mlewand** [comment]
[Responsive toolbar](https://github.com/ckeditor/ckeditor5/issues/416#issuecomment-430246472) has been merged to the master branch 🎉 

![Shrinking toolbar](https://user-images.githubusercontent.com/5353898/66823707-1248fb00-ef47-11e9-8ccf-72955b380e27.png)

**Comment by Reinmar** [comment]
I guess this shows the responsive toolbar a bit better :D 

![Oct-18-2019 12-15-10](https://user-images.githubusercontent.com/156149/67086428-fe98d100-f1a0-11e9-95bc-403dc632317e.gif)

**Comment by zadam** [comment]
Documentation links on the release page for 15.0.0 return 404:

![grafik](https://user-images.githubusercontent.com/617641/67389560-25894580-f59b-11e9-977f-1fd511eadc3d.png)

**Comment by ma2ciek** [comment]
> Documentation links on the release page for 15.0.0 return 404:

Hi, we're in the middle of the release process, stay tuned, docs should be updated soon.

**Comment by mlewand** [comment]
@zadam thanks for letting us know. The docs will be published today, sorry for inconvenience.

**Comment by mlewand** [comment]
This iteration is done! A new one has started - it includes code blocks, table cell styling, restricted editing, special characters and more!

See [iteration 28](https://github.com/ckeditor/ckeditor5/issues/5653) roadmap for a complete overview!
